### PR TITLE
Update request-response.rst for response file

### PR DESCRIPTION
**Pull request by kicaj**

### CODE REVIEW

File: en/controllers/request-response.rst [hunk 1]
```diff
@@ -746,9 +746,7 @@ ics generated on the fly from a string::
 Callbacks can also return the body as a string::
```
Comment (othercorey) [inline en/controllers/request-response.rst]: This section should be removed or updated to show how to use callback using `withBody()`.

File: en/controllers/request-response.rst [hunk 2]
```diff
@@ -746,9 +746,7 @@ ics generated on the fly from a string::
...
 
     $path = '/some/file.png';
-    $this->response->body(function () use ($path) {
-        return file_get_contents($path);
-    });
+    $this->response->withStringBody(file_get_contents($path));
```
Comment (markstory) [inline en/controllers/request-response.rst]:
```suggestion
    return $this->response->withStringBody(file_get_contents($path));
```

We should indicate that this method will return  a new instance with `$response =` or `return`.

**Review by Arhell — COMMENTED** [review]
@kicaj  please resolve conflicts

**Comment by markstory** [comment]
There are no changes here anymore. Did a rebase/conflict resolve go poorly?

**Comment by ndm2** [comment]
The section that was initially changed [has meanwhile been removed](https://github.com/cakephp/docs/pull/6957/files).

**Pull request closed without merging**